app/mobile/src/App.tsx

```diff
@@ -0,0 +1,54 @@
+import { StatusBar } from "expo-status-bar";
+import React, { useState } from "react";
+import { StyleSheet, Text, View, Image, TouchableOpacity } from "react-native";
+import LandingScreen from "./components/LandingScreen";
+import StorybookUIRoot from "./storybook";
+
+export default function App() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity
+        onPress={() => setIsOpen(!isOpen)}
+        style={{
+          position: "absolute",
+          top: 30,
+          right: 10,
+          backgroundColor: "red",
+          zIndex: 99,
+          borderRadius: 25,
+        }}
```

Comment:
We probably want to avoid from doing inline styles like this with ad-hoc values, as it's gonna introduce a tonnes of inconsistency with magic values all over the place, and prefer using values from a theme of defined tokens similar to how it's done on web.

Maybe one to address once we sort out UX stuff more formally in the app though? Since I don't think MUI we use on web can be used here since it's dependent on web primitives.

Reply:
As this is the X button for storybook I think it's ok for now? I don't want to overengineer this, and as we have ticket #2015 above this will actually disappear before we get to any kind of production-ready stage.